### PR TITLE
Setup indentation 4

### PR DESCRIPTION
**Pull request by AzisK**
Additionally setup warning for double quotes

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,97 +1,108 @@
 import express from 'express';
 import bodyParser from 'body-parser';
 import cors from 'cors';
-import { graphiqlExpress, graphqlExpress } from 'apollo-server-express';
-import { snakeCase } from 'lodash';
+import {
```
Comment (ignasposka) [inline src/index.ts:4]: IMHO this looks much worse than a straight line as before. Maybe we could allow few characters more to preserve such imports inline? Previous line is not that long I think 🤔

File: src/schema/resolvers/animal.ts [hunk 2]
```diff
@@ -1,141 +1,185 @@
...
 const resolvers: IResolvers = {
-  Query: {
-    animals: async (_, __, { pgClient }) => {
-      const dbResponse = await pgClient.query(getAnimalsQuery());
-      return dbResponse.rows;
-    },
-    animal: async (_, { id }, { pgClient }) => {
-      const dbResponse = await pgClient.query(getAnimalQuery(id));
-      return dbResponse.rows[0];
+    Query: {
```
Comment (ignasposka) [inline src/schema/resolvers/animal.ts:30]: Same here, looks much less readable when split into multiple lines 😖

**Review by ignasposka — CHANGES_REQUESTED** [review]
Requesting changes, because in some places readability looks much worse at least for me. If you think opposite let's discuss.

File: src/index.ts (hunk 1 above)
Comment (AzisK) [reply src/index.ts:4]: I used a VS Code plugin 'Beautify' to auto-reindent all the lines. It has also did this "beauty" according to him. I did not find this important to bring it back to one line. If you think that is important, I can bring it back to the old version

File: src/schema/resolvers/animal.ts (hunk 2 above)
Comment (AzisK) [reply src/schema/resolvers/animal.ts:30]: I used a VS Code plugin "Beautify" to auto-reindent all the lines. It has also did this "beauty" according to him. I did not find this important to bring it back to one line. If you think that is important, I can bring it back to the old version

File: src/index.ts (hunk 1 above)
Comment (ignasposka) [reply src/index.ts:4]: I think you should adjust `Beautify` settings to allow longer lines, or just bring the old formatting back.

File: test/user.grapqhl.test.ts [hunk 3]
```diff
@@ -23,105 +24,126 @@ describe ('GraphQL user integration tests', () => {
...
             .send({
-                query: `{ users ${userFields} }`
+                query: `{ users ${userFields} }`,
             })
             .expect(200)
             .set('Authorization', `Bearer ${process.env.BEARER_TOKEN}`)
             .end((err, res) => {
                 if (err) return done(err);
-                const { body: { data: { users } } } = res;
+                const {
```
Comment (ignasposka) [inline test/user.grapqhl.test.ts:37]: This multiline split also looks unnecessary.

**Review by ignasposka — APPROVED** [review]
Just fix all places such I noted in my comment and you're good to merge.

File: test/user.grapqhl.test.ts (hunk 3 above)
Comment (AzisK) [reply test/user.grapqhl.test.ts:37]: It does not seem possible with prettier, so I will look into other formatters

**Review by psaulenas — APPROVED** [review]

**Pull request merged**